### ISSUE TITLE
distill result on ImageNet

### ISSUE THREAD

**Issue by as754770178** [NervanaSystems/distiller#133]
The distill result is well when I distill model in small dataset, but when I distill `resnet_v1_50` on ImageNet, the result is not good. The top-5 of teacher model is 92.7% , but top-5 of student model is 92%.
Both the teacher model and the student model are `resnet_v1_50`.

**Comment by nzmora** [comment]
Hi @as754770178,

When the teacher and student are the same network, trained with the same dataset, to solve the same problem, you cannot expect the student's accuracy to exceed that of the teacher.  If we could do that, then we could continuously train the same network in a teacher-student setting until it converges to 100% accuracy.  Obviously, this cannot work.
Another way to think about this is to remember that the student is trying to learn from, and mimic, the teacher.  When the teacher and student have the same capacity to learn (model, number of parameters), the student can only learn to be as good as the teacher, not surpass it.  The difference between 92.7% and 92.0% can be due to hyper-parameter values, training stochasticity etc. 

Perhaps try training ResNet18 using ResNet50 and see what happens.
Cheers,
Neta

**Comment by as754770178** [comment]
thanks. Do you have the result of distillation on ImageNet?

**Comment by nzmora** [comment]
Hi @as754770178,

Sorry, no.  If you do use distillation on ResNet50 it'll be interesting to see you results.  Note that for the teacher you will need a model with a higher capacity than ResNet50, such as ResNet101, and that will require time and compute resources.
Good luck,
Neta

**Comment by as754770178** [comment]
I use `pnasnet` which top-5 is 96% distill `resnet_v1_50` that `resnet_v1_50` don't import logits layer, the top-5 of `resnet_v1_50` still low

**Comment by nzmora** [comment]
Closing due to inactivity. Please reopen if needed.
